app.js

```diff
@@ -13,14 +13,27 @@ const onCreateInputField = (todo) => {
     inputElement.classList.add("text");
     inputElement.type = "text";
     inputElement.value = todo.text;
-    inputElement.setAttribute("readonly", "readonly")
+    inputElement.setAttribute("readonly", "readonly");
+
     return inputElement;
 }
+
 const handleDelete = (todo) => {
     todos = todos.filter(todoElement => todoElement.id !== todo.id);
     renderTodos();
 }
 
+const handleEdit = (inputElement, editButton) => {
```

Comment:
Generally event listener functions are prefixed with either **on** or **handle**. Let's choose one and stick with it rather than using them both for consistency.